File: python/code_war/ccard.js
// return masked string
function maskify(cc) {
    var ccc = String(cc);
    var x = "";
    for (var i = 0;i < ccc.length;i++){
        var z =  ccc.length - 4;
        if (i < z){
            x = x + '#';
        }
        else{
            x = x + ccc[i];
        }
    }
    return x;
}

console.log(maskify('464356328793874683568992436579'));

File: python/code_war/ccard.py
def maskify(cc):
    ccc = str(cc)
    mask = ""
    x = 0
    for i in ccc:
        z = len(ccc) - 4
        if x < z:
            mask = mask + '#'
        else:
            mask = mask + ccc[x]
        x = x + 1
    return mask


def maskify2(cc):
    return "#"*(len(str(cc))-4) + str(cc)[-4:]


print(maskify(12345678910))
print(maskify2(12345678910))
